.github/workflows/test.yml

```
@@ -1,11 +1,11 @@
-name: test
-on:
-  pull_request:
-  push: { branches: [ main ] }
-jobs:
-  unit-tests:
-    uses: vapor/ci/.github/workflows/run-unit-tests.yml@reusable-workflows
-    with:
-      with_coverage: false
-      with_tsan: true
-      coverage_ignores: '/Tests/'
+#name: test
+#on:
+#  pull_request:
+#  push: { branches: [ main ] }
+#jobs:
+#  unit-tests:
+#    uses: vapor/ci/.github/workflows/run-unit-tests.yml@reusable-workflows
+#    with:
+#      with_coverage: false
+#      with_tsan: true
+#      coverage_ignores: '/Tests/'
```

No need to disable this; I've updated the reusable workflow to ignore draft pull requests until they're marked ready. But it does require a small tweak to the calling workflow to get the CI run to run automatically when you click the "Ready for Review" button:
```suggestion
name: test
on:
  pull_request: { types: [opened, reopened, synchronize, ready_for_review] }
  push: { branches: [ main ] }
jobs:
  unit-tests:
    uses: vapor/ci/.github/workflows/run-unit-tests.yml@reusable-workflows
    with:
      with_coverage: false
      with_tsan: true
      coverage_ignores: '/Tests/'
```

---

Oh sweet!